### ISSUE TITLE
Docker support wanted?

### ISSUE THREAD

**Issue by rfuehrer** [Toxantron/scrumonline#147]
Hi, I would like to ask if there is still interest in an updated Docker Image? I have prepared something that I would like to provide as PR ;) Or is this a feature to be managed and maintained outside the upstream?

**Comment by rfuehrer** [comment]
Ok, missed https://github.com/chrisns/scrumonline

**Comment by rfuehrer** [comment]
ok, the more I think about it, I still want to ask the question from the issue. What is the attitude of the upstream to an own integration into a docker container? The advantage of this is the support of a CI pipeline, which might be interesting for later tests.

In addition, direct support provides faster onboarding for new developers and users who want to get started with the solution.

Chrisns' approach is an excellent one, but I still believe that an all-in-one package promoted by upstream would allow a much easier start within the sources of Scrumonline. 

Hence the question whether the integration of four files in the default branch is useful and helpful:

- .env
- docker-compose.yml
- Dockerfile
- entrypoint.sh 

Included are currently (and working):
- Apache 2.4
- PHP 7.4
- MySQL 8.0
- phpmyadmin (optional)

The files are so simple that in my opinion there is no need for extensive upstream maintenance and they can be easily maintained by the project. The goal of this approach is to enable the user to start up the solution with just one command.

**Comment by Toxantron** [comment]
Docker support is obviously important and has great community interest, which is why I added Docker a few years ago. However with limited Docker experience and time the image was barely functional and poorly maintained. Options were limited and the image wasn't available anywhere. So when @chrisns first proposed fixes to the Docker image, I was quite happy, but it still meant I was responsible for overseeing and merging the changes. So we agreed on turning his fork into a dedicated "scrumonline docker" project and removed all upstream files in #123.

Reintegrating into into upstream obviously is more comfortable for everyone else, but it kinda takes me back to my previous dilemma of having to maintain it. Since Docker is important, but not essential I prefer keeping the current solution and think you could discuss your ideas with @chrisns.

Let me add, that I appreciate your recent contributions and interest in the project and value your input.I just feel your expectations might be a little to high. I started this pretty much 5 years ago, because I was tired of forgetting the damn paper cards and none of the free solutions really had what we wanted: Good master view for the projector and comfortable mobile view with the card deck. So I said to @dbeuchler "Guess we could write something better after work in 2 weeks". I even chose PHP for the sole purpose of learning it. So we built it, posted it on facebook and it somehow took off, with a peak around April with over 100k users per month. 

**Long story short:** I never intended for this to become a big project, it's a hobby and I try my best to maintain it, but I am quite happy about everything I don't have to take care of. Hope this answers your question.
